custom/enikshay/integrations/nikshay/field_mappings.py

```diff
@@ -150,9 +155,10 @@
 }
 
 health_establishment_type = {
-    'hospital': 'H',
-    'lab': 'L',
+    'Hospital': 'H',
+    'Lab': 'L',
     'pharmacy': 'P',
+    'Doctor': 'P'
```

Comment:
should this have a different letter (same as pharmacy now)

Reply:
Ya it should be the same.